src/app/shared/services/authentication.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+
+import { User } from '../interfaces/user.interface.';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class AuthenticationService {
+
+  constructor() { }
+
+  loggedUser?: User;
+
+  login(user: User): void {
+    this.loggedUser = {...user};
+    localStorage.setItem("token", JSON.stringify(this.loggedUser));
+  }
+
+  logout(): void {
+    localStorage.removeItem("token");
+    this.loggedUser = undefined;
```

Comment:
In such cases it is better to use **null**

Reply:
Ok, got it